lib/commands/init.js

```diff
@@ -78,6 +86,14 @@ module.exports = Command.extend({
       return Promise.reject(new SilentError(`We currently do not support a name of \`${packageName}\`.`));
     }
 
+    // [WIP] ember new foo --lang
```

Comment:
What is the status of this comment? Does it stick around long term? If so, it needs to be tweaked to make a bit more sense (and remove the `WIP` label).

Reply:
It can definitely go! Thanks!